migrate/20230718_pages.rb

```diff
@@ -0,0 +1,12 @@
+# frozen_string_literal: true
+
+Sequel.migration do
+  change do
+    create_table(:page) do
+      column :id, :uuid, primary_key: true, default: Sequel.lit("gen_random_uuid()")
+      column :created_at, :timestamptz
```

Comment:
I think it's not set

```suggestion
      column :created_at, :timestamptz, null: false, default: Sequel.lit("now()")
```
```suggestion
      column :created_at, :timestamptz, null: false, default: Sequel::CURRENT_TIMESTAMP
```